docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1289,12 +1292,32 @@ message PkgInfo {
   string url = 8;
   // Miscellaneous metadata, structured; a way to extend PkgInfo
   repeated StructuredAnnotation structured_annotations = 9;
+  // If set, specifies the properties that the underlying platform should have.
+  // If the platform does conform to these properties, the server should reject
+  // the P4Info when used with a SetForwardingPipelineConfigRequest.
```

Comment:
Nit: use backticks

Reply:
Err, I think I don't know where you mean to use backticks here? I'm not referring to any variable, which is it is used in the rest of the proto e.g.

Reply:
`SetForwardingPipelineConfigRequest`

Reply:
That seems inconsistent with the general style here. Code is backticked, but message names are not. See e.g. https://github.com/p4lang/p4runtime/blob/main/proto/p4/config/v1/p4types.proto#L51-L52.

Reply:
Definitely not totally obvious. In my mind, I suppose it's the difference between a type and an instantiation basically. For a type, I don't use the backticks and for an instantiation of it (like a variable) I do. That said, if I'm quoting a literal code segment rather than talking about the concept, I would also use backticks I suppose. 

That said, if you wanted to go change how it's done elsewhere, I'd be amenable to fixing it here too, even though I find it slightly worse, but I definitely prefer consistency over adding it as this weird one-off.

Reply:
My preference is for consistently putting all code in backticks to keep things simple.
I agree it is not currently done consistently, so if that bothers you, fine to keep as is.

Reply:
My preference is to consistently put all code in back ticks for simplicity.
Agreed that this is not currently done consistently. If that bothers you, fine to keep as is.